src/styles/blocks/review.scss

```diff
@@ -0,0 +1,16 @@
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+  width: 100%;
+
+  .review__text {
+    font-style: normal;
```

Comment:
default style?

Reply:
they asks to go out as different structure from begining. thanks/